sdk/monitor/azquery/example_test.go

```diff
@@ -40,20 +41,38 @@ func ExampleNewMetricsClient() {
 }
 
 func ExampleLogsClient_QueryWorkspace() {
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		panic(err)
```

Comment:
I believe we've been using `log.Fatal()` in the examples.  Personally, I prefer `panic()` as it provides a stack trace so you can see what operation failed.  Take a look at some other examples and do what they do.

Reply:
using the TODO error handling format